spec/content_access_spec.rb

```diff
@@ -587,166 +551,6 @@
     }.to raise_exception(RestClient::BadRequest)
   end
 
-  it "will show up in the owner at distributor on export/import" do
```

Comment:
Should we still verify the manifest includes this information for old Candlepins? What is the plan for SCA distribution?

Reply:
Yes I think we should verify this. The export should still be exporting the content access mode value

Reply:
I believe there is another test that finds the value in the export file. I will check.
This test is no longer valid as it checks the org on the receiving end after import.

Reply:
I did add a test to ensure that the contentAccessMode is still in the export